integration-tests/common/common.go

```diff
@@ -157,12 +159,79 @@ HTTPSPort = 0
 	c.ChainlinkConfig = chainlinkConfig
 	log.Debug().Str("toml", chainlinkConfig).Msg("Created chainlink config")
 
-	envConfig := &environment.Config{NamespacePrefix: "chainlink-ocr-starknet", TTL: c.TTL, Test: t}
-
-	c.Env = environment.New(envConfig)
+	c.Env = &environment.Environment{}
 	return c
 }
 
+// CapturingPassThroughWriter is a writer that remembers
+// data written to it and passes it to w
+type CapturingPassThroughWriter struct {
+	buf bytes.Buffer
+	w   io.Writer
+}
+
+// NewCapturingPassThroughWriter creates new CapturingPassThroughWriter
+func NewCapturingPassThroughWriter(w io.Writer) *CapturingPassThroughWriter {
+	return &CapturingPassThroughWriter{
+		w: w,
+	}
+}
+
+func (w *CapturingPassThroughWriter) Write(d []byte) (int, error) {
+	w.buf.Write(d)
+	return w.w.Write(d)
+}
+
+// Bytes returns bytes written to the writer
+func (w *CapturingPassThroughWriter) Bytes() []byte {
+	return w.buf.Bytes()
+}
+
+func debug(cmd *exec.Cmd) error {
```

Comment:
made debugging a lot easier. I can remove and replace with qa helper method, but it'd be nice to have something like this to debug on CI and locally